include/groonga/groonga.h

```diff
@@ -980,6 +980,19 @@ grn_obj_set_value(
 #define GRN_OBJ_REMOVE_DEPENDENT (0x01 << 0)
 #define GRN_OBJ_REMOVE_ENSURE    (0x01 << 1)
 
+/**
+ * \brief Remove the specified object from memory.
```

Comment:
サマリーにメモリー上から解放すると入っていると関連ファイルが削除されない雰囲気が醸し出されそうなのでサマリーからは中途半端な詳細は抜きましょう。

```suggestion
 * \brief Remove the specified object.
```

Reply:
fix: [eab8479](https://github.com/groonga/groonga/pull/1916/commits/eab847974d7c22170589a9707b845de3530e9a1f)